spec/classes/puppet_server_config_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'spec_helper'
 
 describe 'puppet::server::config' do
+  before :each do
+    @cacrl = Tempfile.new('cacrl')
+    File.open(@cacrl, 'w') { |f| f.write "This is my CRL File" }
+    Puppet.settings[:cacrl] = @cacrl.path
```

Comment:
This is not a left over it stubs the file function for syncing the MOM CRL to the Compile Masters... While not strictly required it helps with tests.